consensus/replay_test.go

```diff
@@ -142,6 +155,8 @@ LOOP:
 
 		// clean up WAL file from the previous iteration
 		walFile := cs.config.WalFile()
+		fmt.Printf("%s %s\n", walFile, filepath.Dir(walFile))
```

Comment:
no fmt please (or replace with `t.Logf`)

Reply:
oh sure will do

Reply:
debugging leftover, I'll get rid of it